test/navigation/NavigatorSpec.scala

```diff
@@ -76,6 +76,18 @@ class NavigatorSpec extends BaseSpec with ScalaCheckPropertyChecks {
         }
       }
     }
+    "check active  bank account page" - {
+
+      "navigate to how many members" - {
+        "yes is selected" in {
```

Comment:
Add a test for no, even if it is out of scope you have code that naviagates to the how many mambers page when no is selected